bases/world_oecd_public_finance/country_data/table_config.yaml

```diff
@@ -0,0 +1,2591 @@
+      is_in_staging:
+      is_partition:
+    - name: gdp_volume
+      bigquery_type: float64
+      description: GDP, volume
+      temporal_coverage:
+          - 1960(1)2019
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging:
+      is_partition:
+    - name: gdp_potential_current_prices
+      bigquery_type: float64
+      description: GDP, potential, current prices
+      temporal_coverage:
+          - 1985(1)2019
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging:
+      is_partition:
+    - name: gdp_potential_volume
+      bigquery_type: float64
+      description: GDP, potential, volume
+      temporal_coverage:
+          - 1985(1)2019
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging:
+      is_partition:
+    - name: output_gap
+      bigquery_type: float64
+      description: Output gap
+      temporal_coverage:
+          - 1985(1)2019
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging:
+      is_partition:
+    - name: short_term_interest_rate
+      bigquery_type: float64
+      description: Short-term interest rate
+      temporal_coverage:
+          - 1960(1)2019
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit: percent
+      has_sensitive_data: no
+      observations:
+      is_in_staging:
+      is_partition:
+    - name: long_term_interest_rate
+      bigquery_type: float64
+      description: Long-term interest rate on government bonds
+      temporal_coverage:
+          - 1960(1)2019
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit: percent
+      has_sensitive_data: no
+      observations:
+      is_in_staging:
+      is_partition:
+    - name: index_cpi
+      bigquery_type: float64
+      description: Consumer price index
+      temporal_coverage:
+          - 1960(1)2019
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging:
+      is_partition:
+    - name: exchange_rate
+      bigquery_type: float64
+      description: Exchange rate, USD per national currency
```

Comment:
a gente tem no manual de estilos o measurement_unit pra moeda...

Reply:
Mas no caso é uma razão né? usd/moeda local. Não é exatamente a moeda pra ser usada de measurement unit. Pelo que tinha entendido quando falei com o @lucasnascm se não tem a measurement unit na lista do github da BD deixa em branco

Reply:
se não tiver na lista do github e for uma medida padrão a gente tem que incluir, mas nesse caso como é uma razão não tem unidade mesmo